third_party/java/arcs/build_defs/internal/schemas.bzl

```diff
@@ -57,19 +57,23 @@ def arcs_cc_schema(name, src, deps = [], out = None):
         wasm = False,
     )
 
-def arcs_kt_schema(name, srcs, deps = [], test_harness = True):
+def arcs_kt_schema(name, srcs, deps = [], platforms = ["jvm", "wasm"], test_harness = True):
```

Comment:
Given the status of wasm I would argue for making `["jvm"]` the default.

Reply:
> Given the state of wasm I would argue for making `["jvm"]` the default.

In applying this change I discovered a minor bug w/ our Bazel rules: 
If we have just "wasm" as a platform target, it fails to build the `arcs_kt_schema` rule. 

I added a workaround to always build the `jvm` target, even when it's not needed. The source of the problem is in how `arcs_kt_library` is defined.